### DISCUSSION TITLE
weeklies/6

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#124] [Comments]
https://hutusi.com/articles/weekly-6 

 本周刊系列最早是效仿阮一峰做单周分享，但对于我个人来说整理工作量较大，而改成月刊的话周期又太长，且积累素材过多。因此准备改成双周刊形式，每两周的周末发布，记录这段时间里看到的一些有价值的信息。本文为周刊第六期，前期的周刊参见：周刊系列。

**Comment by DeppWang** [comment]
赞一个

**Comment by softempire** [comment]
很不错！

**Comment by hutusi** [comment]
@DeppWang @softempire 谢谢 :-)

**Comment by jccdream** [comment]
赞

**Comment by Mayandev** [comment]
详略得当

**Comment by geektutu** [comment]
六期都看完了，都非常有意思，催更了~

**Comment by hutusi** [comment]
@jccdream @Mayandev @geektutu 感谢支持 :-)

**Comment by eksea** [comment]
赞，希望博主坚持更下去

**Comment by lovelilei** [comment]
截止到上周，有60万人死于新冠病毒（十万分之14），而气候变化造成的四万人数将在2060年达到同等比例，2010年将是这个数字的5倍。

后一个是2100年吗？【挠头】

**Comment by lovelilei** [comment]
很喜欢博主的周刊

**Comment by hutusi** [comment]
@eksea @lovelilei 谢谢支持 :-)

**Comment by hutusi** [comment]
@lovelilei 是2100年，typo fixed 😄
